pkg/ipfailover/keepalived/generator.go

```diff
@@ -165,8 +165,13 @@ func GenerateDeploymentConfig(name string, options *ipfailover.IPFailoverConfigC
 		return nil, err
 	}
 
+	labels := map[string]string{
+		"ipfailover": name,
```

Comment:
What would the value of being more generic?  Name is already a metadata property, and I think it would make more sense to be more specific rather than less.